src/main/java/de/chojo/repbot/config/elements/Badges.java

```diff
@@ -0,0 +1,25 @@
+package de.chojo.repbot.config.elements;
+
+import java.util.Optional;
+
+public class Badges {
+    private boolean enables = true;
+    private String[] badges = new String[0];
+
+    public Badges() {
+    }
```

Comment:
Add a comment which explains why this constructor has an empty body

Reply:
totally for serialization purposes probably. Jackson needs a empty default constructor to create the class.

Reply:
But yeah probably remote it. all other classes only have the default constructor anyway. I'll stuck to this pattern.